src/main/java/teammates/common/datatransfer/FeedbackRubricQuestionDetails.java

```diff
@@ -746,18 +770,24 @@ public String getQuestionResultStatisticsCsv(
         DecimalFormat dfAverage = new DecimalFormat("0.00");
 
         int[][] responseFrequency = calculateResponseFrequency(responses, this);
-        float[][] rubricStats = calculateRubricStats(responses, question);
-        
+        float[][] rubricStats = calculateRubricStats(responses, this);
```

Comment:
I did that at first but I realized it would make the dependency of calculateRubricsStats on calculateResponseFrequency unclear. I will open a separate PR to deal with that.